packages/typography/src/Link/Link.spec.tsx

```diff
@@ -155,4 +155,47 @@ describe('packages/typography', () => {
       expect(div!.tagName.toLowerCase()).toBe('div');
     });
   });
+
+  describe('anchor props', () => {
+    test('href is always passed to rendered component when the prop is supplied', () => {
+      renderLink({ href: 'test' });
+      const div = screen.getByText('Link').parentElement;
+      expect(div).toHaveAttribute('href');
```

Comment:
Why do we expect the div to have the href here?

Reply:
the point is that we are no longer being judicious about what can / cannot receive an href prop if one is passed to the component.